mypy/main.py

```diff
@@ -112,8 +112,11 @@ def process_options() -> Tuple[List[BuildSource], Options]:
             parsed flags)
     """
 
+    # Make the help output a little less jarring.
+    help_factory = (lambda *a, **k:
+                    argparse.RawDescriptionHelpFormatter(max_help_position=28, *a, **k))
```

Comment:
It's unclear to me if this is intended to be a supported use.  From `help(argparse.RawDescriptionHelpFormatter)`: "Only the name of this class is considered a public API. All the methods provided by the class are considered an implementation detail."